debian/.gitignore

```diff
@@ -0,0 +1,8 @@
+.debhelper/
+debhelper-build-stamp
+dosbox-staging.debhelper.log
+dosbox-staging.substvars
+dosbox-staging/
+files
+changelog
+
```

Comment:
Can't we move `debian` to `contrib/linux/debian`? I don't really like it at the root level.

Reply:
Yes! I've now moved it now, and the debian package script simply copies it to root and blows it away as soon as it's done using a bash trap. It's working well and now that pollution is swept firmly under the rug 🧹 👍

Reply:
Nice one @interloper98 👍🏻